pylutron/__init__.py

```diff
@@ -92,7 +98,7 @@ def _send_locked(self, cmd):
     try:
       self._telnet.write(cmd.encode('ascii') + b'\r\n')
     except _EXPECTED_NETWORK_EXCEPTIONS:
-      _LOGGER.exception(f"Error sending {cmd}")
+      _LOGGER.exception("Error sending {}".format(cmd))
```

Comment:
This PR still isn't rebased correctly.  This is showing as a diff here but this is what the current line reads.